src/commands/schema/diff.ts

```diff
@@ -1,17 +1,26 @@
 import SchemaCommand from "../../lib/schema-command";
 import { bold, colorParam, hasColor, reset } from "../../lib/color";
-import { Flags } from "@oclif/core";
+import { Args, Flags } from "@oclif/core";
+
+type Target = "local" | "staged" | "active";
 
 export default class DiffSchemaCommand extends SchemaCommand {
   static flags = {
     ...SchemaCommand.flags,
-    active: Flags.boolean({
-      description:
-        "Compare the local schema to the active schema instead of the staged schema.",
+    text: Flags.boolean({
+      description: "Display the text diff instead of the semantic diff.",
       default: false,
     }),
   };
 
+  static args = {
+    ...SchemaCommand.args,
+    target: Args.string({
+      description: "The target schema to compare against (active | staged).",
+      required: false,
+    }),
+  };
+
   static description =
     "Print the diff between local schema and staged remote schema.";
   static examples = [
```

Comment:
this is probably worth adding ~2 examples, one using a non-default `target` flag, and one using a non-default `diffKind`.